data/json/recipes/recipe_weapon.json

```diff
@@ -6828,5 +6828,130 @@
       [ "scrap", 12]
     ]
   ]
+},{
+  "type" : "recipe",
+  "result": "bayonet",
+  "category": "CC_WEAPON",
+  "subcategory": "CSC_WEAPON_MODS",
+  "skill_used": "fabrication",
+  "difficulty": 6,
+  "time": 360000,
+  "reversible": false,
+  "autolearn": false,
+  "book_learn": [[ "recipe_melee", 5 ]],
+  "qualities":[
+    {"id":"HAMMER","level":3,"amount":1},
+    { "id": "CHISEL", "level": 3 }
+  ], "tools": [
+      [
+      ["tongs", -1]
+      ],
+      [
+      ["anvil", -1]
+      ],
+      [
+      ["swage", -1]
+      ],
+      [
+      ["forge", 300], [ "oxy_torch", 60]
+      ],
+      [
+      ["mold_plastic", -1]
+      ]
+      ],
+  "components":
+    [
+     [
+     [ "steel_lump", 1 ],
+     [ "steel_chunk", 4 ],
+     [ "scrap", 12 ]
+     ],
+     [
+     [ "plastic_chunk", 1 ]
+     ]
+    ]
```

Comment:
Style should be like in newer files. A good example is `data/json/recipes/electronics/parts.json`.